Gems/PhysX/Code/Editor/Source/Components/Conversion/CollidersPrefabConversion.cpp

```diff
@@ -127,6 +128,16 @@ namespace PhysX
             return;
         }
 
+        bool isLevelOpen = false;
+        AzToolsFramework::EditorRequests::Bus::BroadcastResult(
+            isLevelOpen, &AzToolsFramework::EditorRequests::Bus::Events::IsLevelDocumentOpen);
+        if (isLevelOpen)
+        {
+            AZ_TracePrintf("PhysXColliderConversion", "There is a level currently opened in the editor. To run this command please restart the editor and run it before opening any level.\n");
```

Comment:
Maybe worth making it a warning or error to make it more visible in the console that the command failed?

Reply:
I didn't put it for the message when prefabs is disabled... which is a similar case, there is nothing wrong about it, but I get what you mean. Probably not worth another AR run though :)

Reply:
I'll change it for a warning.